### PR TITLE
iPhone Product Landing Page

### PR DESCRIPTION
**Pull request by Orzala-2024**
🔘This is a product landing page which is designed for iPhone products . this repo contains below files:
* landing.html
* styles.css
* readme.md
* linters

Here you can watch a view of my page:

product landing

### CODE REVIEW

File: landing.html
Comment (ahzamir) [inline landing.html:1]: - Your code spacing and alignment are not clean please have a look at your opening tag and closing tags and parent element and child elements for their alignment and spacing.

**Incorrect:**
```html
<!DOCTYPE html>
<html>
    <head>
        <title>Page Title</title>
    </head>
    <body>
<h1>This is a Heading</h1>
        <p>This is a paragraph.</p>
    </body>
</html>
```

**Correct:**
```html
<!DOCTYPE html>
<html>
  <head>
    <title>Page Title</title>
  </head>
  <body>
    <h1>This is a Heading</h1>
    <p>This is a paragraph.</p>
  </body>
</html>
```

**Rule:** Each child element should be indented 2 spaces under its parent element for better readability and maintainability.
**Hint**: Go to the VSCode bottom right corner, you can see "Spaces: 4" (or a similar indicator). Click on that, then click on "Indent Using Spaces" and select 2. You can also use the gray color column lines to help you know where your parent and child elements should be.

Comment (ahzamir) [inline landing.html:1]: - [**OPTIONAL**] - You added a very great design to your project and it looks amazing, try to add some more page content and sections according to the navbar links you added so your product landing page looks better.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,91 @@
...
+
+
+
+
+## Demo 📸
+
+
+- Below you can have a view of my page:
+
+![Technical Documentation page]![picof project](https://github.com/LimaSahibzada/Product-Landing-Page1/assets/167701815/542382ee-8da5-4f5a-9649-ef2cfd524cb1)
```
Comment (ahzamir) [inline README.md]: - Remove one of these `![Technical Documentation page]`, `![picof project]`.
  - Just keep one of them as alt text for your Demo.

**Review by ahzamir — CHANGES_REQUESTED** [review]
### Hi @LimaSahibzada 🙋‍♀️,

> Excellent work! You have done a great job implementing the project requirements but your project still misses some of the requirements:

- Although your code passes the freecodecamp tests, but you still need to work on your project to get the final approval.
  - Check the below inline comments for more details

# Status: Required Changes 🔁
Make changes to your code and commit and request for a re-review.

AlmostThereReachGIF

 ## Highlights 🌞
 
🎯 Great Design implemented.
image

✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed and completed the Product Landing Page and all requirements.
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your project has a professional README.md file
❌Your project code is clean

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Comment by Orzala-2024** [comment]
@ahzamir ,
Kindly review my project.

File: landing.html [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<html lang="en"><html>
+  <meta charset="UTF-8">
+  <meta name=”website” content=”Landing Page”>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (ahzamir) [inline landing.html]: - These codes should be inside your `<head>` tag

File: landing.html [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+<head>
+<title>Iphone Product Landing Page</title>
```
Comment (ahzamir) [inline landing.html]: - Add two space here as title is a child for head tag

File: landing.html [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        <li><a class="nav-link" id="Home" href="#Home">Home</a></li>
+      </nav>
+       <img id="header-img" src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTVFvg0mFEnZPAwxsjURQFk-w0txeaDKQ8CVL2IIaCiyQ&s" alt="logo">
+       <div class="form">
+         <form id="form" action="https://www.freecodecamp.com/email-submit">
+           <label= for="email">
+           <input type="email" id="email" name="email" placeholder="Enter your email address"></input>
+           <input type="submit" id="submit"></input>
+          </form>
+    </header>
```
Comment (ahzamir) [inline landing.html]: - I can not see the closing tag for you div here add it.

File: landing.html [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+    <h1>Simple and Powerful</h1>
+      <p>The iPhone is a line of smartphones produced by Apple .<br>The first-generation iPhone was announced by Jobs!<br></p>
+      <button class="button">Shop Now</button>
+      <center>
+      <iframe id="video" src="https://www.youtube.com/embed/FT3ODSg1GFE?si=0ZlweiXR5__er_ZS" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" referrerpolicy="strict-origin-when-cross-origin" allowfullscreen></iframe></iframe></iframe></center>
```
Comment (ahzamir) [inline landing.html]: - Remove the 2 spaces as they are not a child of your h1

Comment (ahzamir) [inline landing.html]: - Here you have 3 closing tags just for one `<iframe>` solve it.

File: landing.html [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        <div class="footer-icons">
+         <ul>
+           <li><a href="#" target="_blank"><i class="fa fa-linkedin fa-2x"></i></a></li>
+           <li><a href="#" target="_blank"><i class="fa fa-instagram fa-2x"></i></a></li>
+           <li><a href="#" target="_blank"><i class="fa fa-youtube fa-2x"></i></a></li>
+          </ul>
+        <div class="copyright">
+          <p>Designed With Love by Orzala </p>
+      </div>
+    </footer>
```
Comment (ahzamir) [inline landing.html]: - Your 2 divs do not have a closing tags add them

**Review by ahzamir — CHANGES_REQUESTED** [review]
@LimaSahibzada 
- Although you cleaned your code a little bit still there are some issues you need to work on check the below comments for more details:

**Comment by Orzala-2024** [comment]
@ahzamir ,
Kindly review my work. I made some changes.

**Review by ahzamir — APPROVED** [review]
### Hi @LimaSahibzada 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.

- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

- Well Done great design Implemented. 🎯🎉👍

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Product Landing Page project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)
✔️Your code is clean
✔️Your project has a professional README.md file

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

**Pull request merged**